code/go/0chain.net/chaincore/config/config.go

```diff
@@ -39,12 +39,13 @@ func SetupDefaultConfig() {
 	viper.SetDefault("server_chain.block.sharding.min_active_sharders", 100)
 	viper.SetDefault("server_chain.block.sharding.min_active_replicators", 100)
 	viper.SetDefault("server_chain.smart_contract.timeout", 550*time.Millisecond)
+	viper.SetDefault("server_chain.smart_contract.setting_update_period", 300)
```

Comment:
Perhaps use the same default value of `200` as 0chain.yaml has.

Reply:
Perhaps, but I was thinking that maybe we could stagger them, so everything does not happen on the same block?

Reply:
I mean the `setting_update_period` is different from the value of `setting_update_period` in the 0chain.yaml, I think this `300` will be set to the value in the `0chain.yaml` after loading the yaml file.